### ISSUE TITLE
Alternative ENV Dockerfile syntax not warned

### ISSUE THREAD

**Issue by Fixmetal** [hadolint/hadolint#752]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

As stated by [Dockerfile ENV](https://docs.docker.com/engine/reference/builder/#env) reference the Alternative syntax
```
ENV MY_VAR my-value
```

is `supported for backward compatibility, but discouraged for the reasons outlined above, and may be removed in a future release.`
### Expected behavior

Given the following Dockerfile:
```
FROM debian:bullseye-slim

ENV MY_VAR my-value

RUN echo $MY_VAR
```

hadolint should generate *at least* a warning quoting the Dockerfile reference. Something like:
```
$ hadolint Dockerfile
Dockerfile:3 DLXXXX warning: Do not use ENV alternative syntax, see https://docs.docker.com/engine/reference/builder/#env
```

### Actual behavior

hadolint just doesn't warn anything.

### Steps to reproduce the behavior

```
$ docker run -it -v $PWD/Dockerfile:/tmp/Dockerfile --entrypoint hadolint --rm hadolint/hadolint /tmp/Dockerfile
$ echo $?
0
$ docker run --rm hadolint/hadolint hadolint --version
Haskell Dockerfile Linter v2.8.0-0-g398770f-dirty
```

```Dockerfile
FROM debian:bullseye-slim

ENV MY_VAR my-value

RUN echo $MY_VAR
```
